lib/generators/administrate/install/install_generator.rb

```diff
@@ -40,6 +40,11 @@ def warn_about_invalid_models
           puts "WARNING: Unable to generate a dashboard for #{invalid_model}."
           puts "         It is not connected to a database table."
         end
+
+        unnamed_constants.each do |invalid_model|
+          puts "WARNING: Unable to generate a dashboard for #{invalid_model}."
+          puts "         It is not a valid Ruby constant."
```

Comment:
I spent a bit thinking about this. Does this even need to be a warning? I feel like we can pretty certainly say that if a class isn't assigned to a constant, you can ignore it. (Also you should look at the implementation of `descendants` if you haven't in the past, because it's hilarious)